### ISSUE TITLE
From this morning fadblock don't work anymore

### ISSUE THREAD

**Issue by AleMingh** [0x48piraj/fadblock#182]
Fadblock don't work anymore on Vivaldi and Edge on Windows 10 and Linux KDE Neon
This happened since this morning.

**Comment by JTH366** [comment]
Fadblock has stopped working completely in Chrome also, anyone, any ideas of what the issue may be?

**Comment by pquodling** [comment]
If you read issues (at least the last few) before posting, you would see that this is already reported, and a fix identified - just waiting for it to be rolled into operation

**Comment by 0x48piraj** [comment]
The latest YouTube patch has not been deployed globally yet, so the previous version continues to function for the time being.

I've made updates to the backend, so if anyone encounters issues with the current version, they can simply update their installed FadBlock.

Here's how:

![image](https://github.com/0x48piraj/fadblock/assets/5800726/0ff837f1-7b1e-4da6-af6d-48b786e881e8)

**Comment by direraven1143** [comment]
You are doing amazing work @0x48piraj

**Comment by AleMingh** [comment]
I tried to update FADBock as you showed but after this try if I caome back to the FADBlock menu, it say again I need to update. 
Same behavior on Vivaldi on linux and Edge on Windows 10.
I also tried to reinstall FAD, restart the browser after the installation but nothing solve the update and work problem on youtube

**Comment by DigitalBear70** [comment]
Same experience as AleMingh.  I'm using EDGE with Windows 10.

**Comment by Bale626** [comment]
I'm using Firefox on Windows 11, can confirm it is not currently blocking the ads.

**Comment by 0x48piraj** [comment]
The patch is constantly changing, I will have to push a new release, give me this weekend you guys.

**Comment by 0x48piraj** [comment]
Can you guys update the extension again?

![image](https://github.com/0x48piraj/fadblock/assets/5800726/b32435b8-22fd-4284-b1f2-e0cbabc7f923)

Just clicking on "Update" will do, it'll update in a blink and will say "Updated!". Just close the menu and you can resume watching YT again. Let me know if this patch works.

**Comment by DigitalBear70** [comment]
I just checked a handful of videos on YouTube and the extension seems to be working again.

Thank you so, so much.  You are the **GOAT**.

**Comment by LiN-Taiwan** [comment]
> Can you guys update the extension again?
> 
> ![image](https://private-user-images.githubusercontent.com/5800726/323829594-b32435b8-22fd-4284-b1f2-e0cbabc7f923.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1MzQyNzYsIm5iZiI6MTcxMzUzMzk3NiwicGF0aCI6Ii81ODAwNzI2LzMyMzgyOTU5NC1iMzI0MzViOC0yMmZkLTQyODQtYjFmMi1lMGNiYWJjN2Y5MjMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTlUMTMzOTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTBhOTMwODljZTFiZmE1ZGRhY2MyNWRiNGQyZDgzODE5ZGMwNDkzMGFmZGQ4NGUzYzVmMjZmOTI1ZTk1OTM3ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.fZ3BzPRbjlwV8-VNY5zTSn-jzNTNaTBiCC0y72yLE8I)
> 
> Just clicking on "Update" will do, it'll update in a blink and will say "Updated!". Just close the menu and you can resume watching YT again. Let me know if this patch works.

How can I  update the extension？ I don't see the "Update" button.😖

**Comment by AleMingh** [comment]
After updating the extension, on Linux KDE Neon with Vivaldi, everything works perfectly again.
Thanks

**Comment by AleMingh** [comment]
> > Can you guys update the extension again?
> > ![image](https://private-user-images.githubusercontent.com/5800726/323829594-b32435b8-22fd-4284-b1f2-e0cbabc7f923.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1MzQyNzYsIm5iZiI6MTcxMzUzMzk3NiwicGF0aCI6Ii81ODAwNzI2LzMyMzgyOTU5NC1iMzI0MzViOC0yMmZkLTQyODQtYjFmMi1lMGNiYWJjN2Y5MjMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTlUMTMzOTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTBhOTMwODljZTFiZmE1ZGRhY2MyNWRiNGQyZDgzODE5ZGMwNDkzMGFmZGQ4NGUzYzVmMjZmOTI1ZTk1OTM3ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.fZ3BzPRbjlwV8-VNY5zTSn-jzNTNaTBiCC0y72yLE8I)
> > Just clicking on "Update" will do, it'll update in a blink and will say "Updated!". Just close the menu and you can resume watching YT again. Let me know if this patch works.
> 
> How can I update the extension？ I don't see the "Update" button.😖 How can I go to this page?

On the extension select the burger menu (3 dot on top right) and select "Update". 
After update I suggest to refresh the youtube page.

**Comment by 0x48piraj** [comment]
Phew! I will soon release the next version which will probably have the capability to adapt to geo-specific patches so we can minimize the downtime even more.

Sorry for the delay, you guys. I wasn't able to push any updates while the previous patch remained effective worldwide, including in my location.

I will be closing duplicate issues while redirecting some to this thread for now, even if there's any problem with the new update, feel free to reach out!

**Comment by Farseer123456789** [comment]
Ad blocker has stopped working for Firefox.  Cannot update the extension either

**Comment by christian100kodehode** [comment]
Was not working, then i check here and update from the menu and it works as intended again... You rock 0x48piraj! (On Firefox.)

**Comment by Frammer** [comment]
> Phew! I will soon release the next version which will probably have the capability to adapt to geo-specific patches so we can minimize the downtime even more.
> 
> Sorry for the delay, you guys. I wasn't able to push any updates while the previous patch remained effective worldwide, including in my location.
> 
> I will be closing duplicate issues while redirecting some to this thread for now, even if there's any problem with the new update, feel free to reach out!

It seems to not be working on Chrome anymore. Running the update a few days ago did work, but G has out-updated you it seems. I look forward to the next release which may fix it.

**Comment by direraven1143** [comment]
Same - stopped working for me on Edge this morning despite updating. Thanks for continuing to try!

**Comment by CommicKick69ing** [comment]
For chrome (at least for me) it skips the initial ads at the beginning of the video, but not the mid roll ads.

**Comment by direraven1143** [comment]
Actually mine is exactly the same as @CommicKick69ing on Edge

**Comment by 0x48piraj** [comment]
Yeah, I have been also getting some issues, a few ads are going through while others are getting skipped as intended, seems like YT is dynamically applying patches?

I will investigate further and put out the new version this week. Hold on, everyone!

**Comment by DigitalBear70** [comment]
JAHU

I'm afraid that _FadBlock_ is not working anymore for me either.  I'm getting ads at the beginning and mid way through videos.  

Windows 10, EDGE Browser.  Version 124.0.2478.51 (Official build) (64-bit)

We know that you're working hard to fix it.  It's appreciated.  TY

**Comment by 0x48piraj** [comment]
I think I have found the culprit and a new version is being tested right now in a closed loop, if all goes well, the new update will drop before tomorrow's end.

**Comment by MrDrDevil** [comment]
Thank you so much 0x48piraj
you are awesome

**Comment by CommicKick69ing** [comment]
The website cookies doubleclick.net and tpc.googlesyndication.com seem to responible for ads in some way since blocking those sites using EditThisCookie stopped the ads

**Comment by DigitalBear70** [comment]
Thank you **CommicKick69ing** for your suggestion.    

"The website cookies _doubleclick.net, ad.doubleclick.net, tpc.googlesyndication.com_ seem to responsible for ads in some way since blocking those sites using _**EditThisCookie**_ stopped the ads".

So far, I've loaded about a dozen videos on **YouTube** after doing this and no ads.

_Sanity preserved._

**Comment by 0x48piraj** [comment]
THE NEW UPDATE WAS JUST PUSHED OUT! (It takes Chrome Dev team to check the version for safety before it gets pushed, just keep FadBlock Origin until it gets automatically updated to `1.1`)

And I have added a secondary slot for redundancy so it can handle geo-specific patches, it should work now.

**Comment by DigitalBear70** [comment]
THANK YOU PIYUSH!!

**Comment by Grg76** [comment]
I hope the Firefox addon will also get the update.

**Comment by 0x48piraj** [comment]
I was testing the version all night and the results we got were so far so good. I am 80% sure this patch will work but if it doesn't, don't panic, I am working on it actively.

In a few updates, FadBlock Origin will again be stable as it was before.

Yes @Grg76, just packaging the update, it will be pushed under 30 minutes.

**Comment by AverageYTuser** [comment]
I know the Update is on the way but I just wanted you to know this. 
I removed Fadblock origin and re-added it and I feel like it might worsen its function, hopefully it actually doesn't. anyways looking forward to the Upcoming update tomorrow or on the next coming days

**Comment by 7Aliens** [comment]
Hallo ich will ja nicht haten aber da geht garnix zu 80%.. jedes Video Werbung.. Habe Chrome

**Comment by liftacar** [comment]
Will it automatically update the plug in or do we have to manually do it?

**Comment by CptNautilus** [comment]
Since this morning, Fadblock is no longer blocking the ads on Chrome. It still does on Brave though.

**Comment by aestetix** [comment]
> THE NEW UPDATE WAS JUST PUSHED OUT! (It takes Chrome Dev team to check the version for safety before it gets pushed, just keep FadBlock Origin until it gets automatically updated to `1.1`)
> 
> And I have added a secondary slot for redundancy so it can handle geo-specific patches, it should work now.

Can you also make the update available on the repo? I see two branches, one called "json" that was updated recently, but I do not see how they work together.

**Comment by Meza85** [comment]
stopped blocking ads again.. since now. was fine yesterday.

**Comment by jacanteros** [comment]
Stopped working with Chrome too.
<img width="388" alt="Captura de pantalla 2024-04-26 a las 13 15 53" src="https://github.com/0x48piraj/fadblock/assets/62242787/37509ab6-336a-4683-9f6d-54a4dd5b2744">
I suspect it has to do with this... (Improved safe browsing do not trust this extension.)

**Comment by b940315** [comment]
FAD didn't work; both the opening and in-video ads are. 
I tried updating it.
From Taiwan.

**Comment by 0x48piraj** [comment]
@liftacar, no, it will auto-update as said earlier. Please wait till v1.1 drops, Chrome takes like 48 hours to process the updates.

@jacanteros, Chrome has marked every YT blocker - everything was getting flagged left and right a couple of months ago, you can find numerous reddit threads surrounding this.

**Comment by D0B0Y** [comment]
Stopped working for me completely a few days ago on firefox, I un-installed and re-installed and getting zero blocks in the counter , tried to update and still nothing. is this a geo location thing?

**Comment by MJPeters82** [comment]
Still not working on Chrome. I've tried updating the extension multiple times over the last couple days, and still get ads.

**Comment by Derara24** [comment]
fadblock not working on chome

**Comment by n1mrod88** [comment]
I'm using FireFox and FBO still doesn't work.

**Comment by ene0s** [comment]
Same, not working on Chrome since a few days. Thanks for your work along the months, I really like the extension.

**Comment by liftacar** [comment]
Fadblock not working on Chrome in the US as of 4/27/24 11 am EST

**Comment by ckeene159** [comment]
Fadblock not working on Firefox in the US as of 4/27/24 12:30 p.m. CDT
clicked on "Update", got "Updated" but still doesn't work
first noticed last night, but we hadn't watched youtube for about a week before that
I see comments above that update has been pushed, may take a couple days to come through.  I hope it will work again, we love it and we donated!

**Comment by Farseer123456789** [comment]
Stopped working for Firefox

**Comment by Shaimlesss** [comment]
Hello, it's not working on Opera either, i've been blocking the ads by hand. it's quicker then waiting for 2 ads to run through (seems like they have new unskipable ads)

**Comment by DictionaryManHK** [comment]
Stopped working on Google Chrome on Version 124.0.6367.78 (64-bit), Windows 10 version 22H2 build 19045.4291
Those ads seem to be around 10 seconds, skip button was offered at 5 seconds mark. 
thank you for your continuous support of this project

**Comment by OneWingedAnge7** [comment]
Stopped working on Brave too :(

**Comment by AverageYTuser** [comment]
As of 29 April, Philippine time. It is still not working on Opera gx browser

**Comment by Grg76** [comment]
On Firefox, the 2.7 update is out. and for me it is working again. However some ads are not skipped, just stopped, and i need to press the play button to skip the ad and play the video. But thats not a big problem for me, better that watch the ads :)

Thank you Piyush :)

**Comment by AverageYTuser** [comment]
Hey Piyush,
I think I have something that might fix the problem of fadblock. I have a PC and a work Laptop, my laptop's Fadblock still works because If I recall correctly, I did not update it, while I was using my PC I panicked when the Ads started playing again so I deleted it and redownloaded it, out of nowhere it ain't working no more, Update still has not yet came. 

TLDR: I ignored the fadblock updates on my Laptop and it still surprisingly works
Hopefully this helps.

**Comment by NoUserNameIWantIsAvailable** [comment]
> > the extnension hast updated to 1.1 version on chrome web store, its been around 4 days since the owner sad the he will update the extension (or chrome) as of april 30
> 
> really? its still v1.0 from my view now

it's still 1.0 on the chrome extension webstore. i'm on chrome, ads are still happening as of 4/30/24 at 11:38am EDT.

**Comment by username1871** [comment]
I'm on Brave/chrome updated but is still on v1.0

tried re-install still v1.0 as stated on the chrome extension store

**Comment by 0x48piraj** [comment]
![image](https://github.com/0x48piraj/fadblock/assets/5800726/0ada35de-1990-4f24-bdd5-60e54dd770f4)


My hands are tied as of now guys, all I can say is to wait a little bit more.

**Comment by Logical-Vanilla** [comment]
For those following this issue who haven't seen, the owner did push the update last week, chrome just hasn't updated the extension yet.

(referencing issue 187)
https://github.com/0x48piraj/fadblock/issues/187#issuecomment-2088248089

**Comment by Zjefken** [comment]
Google probably shadow banned your extension from updating because it blocks ads on youtube or something

**Comment by MJPeters82** [comment]
> Google probably shadow banned your extension from updating because it blocks ads on youtube or something

That's what I've been thinking. Google does own YouTube.

**Comment by heitorpfigueira** [comment]
Could there be a way to install the updated extension without going through Google Web Store? Like an executable, dll or something similar, even if it requires manual steps. I'm not sure if you gain something from using the store but I'm afraid they are blocking your stuff.

**Comment by ckeene159** [comment]
Now working with Firefox.  In the US as of 5/01/24 10 p.m. CDT

**Comment by 0x48piraj** [comment]
The Chrome just pushed the extension. The extension will auto-update in 24 hours for everyone who has it - for those who want it instantly can uninstall and install again - beware - you will have to login again if you supported the extension.

**Comment by direraven1143** [comment]
Glorious - works again on Edge, amazing!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by direraven1143** [comment]
Glorious works again on Edge :)

**Comment by JTH366** [comment]
> Fadblock has stopped working completely in Chrome also, anyone, any ideas of what the issue may be?

As of this morning, Fadblock is again blocking Ads on YouTube. Thank you 0x48Piraj

**Comment by seebeedub** [comment]
V1.1 appeared for Chrome and the ads are zipping by again just how I like them. Just enough to know the creators are getting paid, Google is getting paid, and the ads go by like shooting stars. It's a win, win, win!

**Comment by AverageYTuser** [comment]
The Ads are being sweeped again as of right now, it's finally working on Opera gx browser, Wishing you more success in the future. Love from the Philippines.

**Comment by 0x48piraj** [comment]
Hopefully from now on, we can just push the in-app "Update" button to publish patches so to never get a _long and painful_ downtime like this. Fingers crossed.

Cheers everyone and thank you all for the support! I couldn't have done any of it without y'all! 🍾

**Comment by funglol718** [comment]
Hey just spotted a minor thing that sometimes the videos with skipped ads are muted. Dont know if thats a youtube bug of mine or anyone else experiencing same thing.

**Comment by Shaimlesss** [comment]
Hello, i haven't noticed that yet but thanks for the headsup.

**Comment by 0x48piraj** [comment]
@funglol718 I will look into that, it's a reported bug but it's not reproducible so fixing it is very hard. Anyways, the original issue served its purpose so closing this one.
